src/containerapp/azext_containerapp/tests/latest/utils.py

```diff
@@ -7,9 +7,30 @@
 import requests
 from azext_containerapp.tests.latest.common import TEST_LOCATION
 from azure.cli.core.azclierror import MutuallyExclusiveArgumentError, RequiredArgumentMissingError, InvalidArgumentValueError
+from msrestazure.tools import parse_resource_id
+
 from azure.cli.testsdk import (JMESPathCheck)
 
 
+def prepare_containerapp_env_for_app_e2e_tests(test_cls):
+    from azure.cli.core.azclierror import CLIInternalError
+    from .common import TEST_LOCATION
+    rg_name = f'client.env_rg_{TEST_LOCATION}'.lower().replace(" ", "").replace("(", "").replace(")", "")
+    env_name = f'env-{TEST_LOCATION}'.lower().replace(" ", "").replace("(", "").replace(")", "")
```

Comment:
If we run tests in parallel (recording tests and live tests) and different test cases use the same rg_name and env_name, will there be test failures?

Reply:
For testing app scenarios, let them use the same env in the same region to test without having to create a new environment every time.

Reply:
Different test app has different app name, so they can pass without conflict.

Reply:
This will result in a remaining resource group that stores env. 
Let's merge this PR first, I will consider deleting the resource group in our custom cleanup script.